.storybook/main.js

```diff
@@ -1,44 +1,26 @@
-const path = require('path')
-
-module.exports = {
+/** @type { import('@storybook/svelte-vite').StorybookConfig } */
+const config = {
   "stories": [
     "../**/*.stories.mdx",
     "../**/*.stories.@(js|jsx|ts|tsx|svelte)"
   ],
-  "addons": [
+  addons: [
```

Comment:
Why is this one unquoted but the others still have quotes?

Reply:
it only needs the quotes when there are special characters in the strings, so I will remove them.